src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -62,8 +65,12 @@
     private static final List<String> THIRD_AUTHOR_GLOB_LIST = Arrays.asList("**[!(.md)]", "collated**");
     private static final List<String> FOURTH_AUTHOR_GLOB_LIST = Collections.singletonList("collated**");
 
-    private static final List<Format> CONFIG_FORMATS = Format.convertStringsToFormats(Arrays.asList(
+    private static final List<FileType> CONFIG_FORMATS = FileType.convertStringFormatsToFileTypes(Arrays.asList(
             "java", "adoc", "md"));
```

Comment:
Consider making it singletonList?

Reply:
Not possible? As the resulting CONFIG_FORMATS will have more than one elements inside

Reply:
I mean instead of `Arrays.asList `, use `Collections.singletonList`

Reply:
![image](https://user-images.githubusercontent.com/35756209/62278652-16529a00-b47b-11e9-802b-e4f4ddf61cf8.png)

Only possible to use singletonList if you are parsing in one item inside to become a list. But I'm parsing in 3 items

Reply:
OK, sorry for the inconvenience caused.